### ISSUE TITLE
Too old for the modern Android

### ISSUE THREAD

**Issue by igsol** [jp-bennett/Fwknop2#68]
Google play doesn't allow installation neither on Android 11 nor 12.
Could it be possible to build package compatible with the modern Android versions?

**Comment by DWDuq** [comment]
I agree.  Just upgraded and missing being able to use the app.  Did you find an alternative?

**Comment by igsol** [comment]
> Did you find an alternative?

No.

**Comment by dugite-code** [comment]
@DWDuq @igsol I'm using termux the the fwknop cli client to get around the lack of android app atm.

using the --save-rc-stanza `fwknop -A tcp/22 -a 192.168.0.10 -D example.tld --key-base64-rijndael=$key --key-base64-hmac=$hmac --save-rc-stanza`

and then a simple shell script with tasker to call the cmd

```
#!/data/data/com.termux/files/usr/bin/bash

fwknop -a 192.168.0.10 --rc-file ~/.fwknoprc
```

**Comment by sshaikh** [comment]
@dugite-code How do you get/install the cli client in Android?

**Comment by weini22** [comment]
Since mid of June 2025, there is a new set of GUI tools including a new Android client:
https://portguard.net/docs/gui-tools/client-tools

This site links to a forked github repository:
https://github.com/yangcancai/fwknop

But there is no source code available for the new client tools, at least I was not able to find it.
Does anybody have an idea, how to check the new Android APK for backdoors?

**Comment by weini22** [comment]
Interesting discussion here:
https://github.com/mrash/fwknop/issues/376
